tdrs-frontend/Dockerfile

```diff
@@ -2,6 +2,8 @@
 # Stage 0: Create the app directory, install dependencies, and copy in the source code
 FROM node:16.13-alpine3.14 AS localdev
 
+RUN apt-get update
+RUN apt-get install -y libgbm-dev
```

Comment:
nit: add a comment about how this is needed for the pa11y cli

Reply:
I am reverting it because this is not where I had to add it, we don't seem to be using the docker container in CI

Reply:
Ah right right